app/controllers/v0/profile/personal_informations_controller.rb

```diff
@@ -39,21 +39,15 @@ def raise_error!
       end
 
       def log_errors_for(response)
-        if response.gender.nil? || response.birth_date.nil?
-          sanitized_response = sanitize_data(response.to_h)
-          sanitized_params = sanitize_data(params)
-          log_message_to_sentry(
-            'mpi is missing data bug:',
-            :info,
-            {
-              response: sanitized_response,
-              params: sanitized_params,
-              gender: response.gender,
-              birth_date: response.birth_date
-            },
-            profile: 'pciu_profile'
-          )
-        end
+        return unless response.gender.nil? || response.birth_date.nil?
+
+        Rails.logger.error("mpi missing data: #{I18n.t('common.exceptions.MVI_BD502.detail')}", {
+                             response:,
+                             params:,
```

Comment:
Are these not being sanitized anymore? What's in `params`? I worry there's something in there that, combined with `birth_date` below, could constitute  PII.

Reply:
That's a good callout @rmtolmach, we should continue sanitizing this information. I will add that back ASAP.